core/src/main/java/com/datastax/oss/driver/api/core/tracker/RequestTracker.java

```diff
@@ -36,38 +37,147 @@
 public interface RequestTracker extends AutoCloseable {
 
   /**
-   * @deprecated This method only exists for backward compatibility. Override {@link
-   *     #onSuccess(Request, long, DriverExecutionProfile, Node, String)} instead.
+   * Invoked each time a request succeeds.
```

Comment:
I have reordered methods in the interface, so that most recent versions of all distinct methods are placed first. This shall give a better view on their current state.